docs/static/releasenotes.asciidoc

```diff
@@ -1,4 +1,94 @@
 [[releasenotes]]
 == Release Notes
-coming[9.0.0]
 
+* <<logstash-9-0-0-beta1,Logstash 9.0.0-beta1>>
+
+[[logstash-9-0-0-beta1]]
+=== Logstash 9.0.0-beta1 Release Notes
+
+==== Breaking changes
+* Default buffer type to 'heap' for 9.0 https://github.com/elastic/logstash/pull/16500[#16500]
+* Remove `http.*` settings https://github.com/elastic/logstash/pull/16552[#16552]
+* Remove the Arcsight module and the modules framework https://github.com/elastic/logstash/pull/16794[#16794]
+* Remove `event_api.tags.illegal` for 9.0 https://github.com/elastic/logstash/pull/16461[#16461]
+* Remove deprecated modules netflow, fb_apache and azure https://github.com/elastic/logstash/pull/16514[#16514]
+* Remove ingest-converter https://github.com/elastic/logstash/pull/16453[#16453]
+* Drop JDK 11 support https://github.com/elastic/logstash/pull/16443[#16443]
+* Remove enterprise search from default distribution https://github.com/elastic/logstash/pull/16818[#16818]
+* Enable `log.format.json.fix_duplicate_message_fields` by default https://github.com/elastic/logstash/pull/16578[#16578]
+
+==== New features and enhancements
+* Add `%{{TIME_NOW}}` pattern for sprintf https://github.com/elastic/logstash/pull/16906[#16906]
+* Introduce a new `xpack.monitoring.allow_legacy_collection` flag to explicitly permit legacy monitoring https://github.com/elastic/logstash/pull/16586[#16586]
+* Use UBI as base image https://github.com/elastic/logstash/pull/16599[#16599]
```

Comment:
This might also be better placed in the `breaking changes` doc - any docker images based from the Logstash docker image might break if they are relying on using `apt-get` to pull new packages